### PR TITLE
feat : Merge meeds-io/MIPs#105

### PR DESCRIPTION
**Pull request by rdenarie**
This commit add new connector gamification exoplatform. In addition, it removes some old css which should not be used, and which make some visual issues

### CODE REVIEW

**Review by azayati — APPROVED** [review]
LGTM

**Review by mkrout — APPROVED** [review]

File: commons-exo-extension/src/main/webapp/WEB-INF/gatein-resources.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="ISO-8859-1" ?>
+<!--
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2020 Meeds Association
```
Comment (azayati) [inline commons-exo-extension/src/main/webapp/WEB-INF/gatein-resources.xml]: wrong header

**Review by azayati — CHANGES_REQUESTED** [review]

File: commons-exo-extension/src/main/webapp/WEB-INF/gatein-resources.xml (hunk 1 above)
Comment (rdenarie) [reply commons-exo-extension/src/main/webapp/WEB-INF/gatein-resources.xml]: done

**Review by azayati — APPROVED** [review]
LGTM

File: commons-exo-extension/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "extends": [
+    "eslint-config-meedsio"
+  ],
+  "globals": {
+    "gapi": true,
+    "webConferencing": true
```
Comment (ahamdi) [inline commons-exo-extension/.eslintrc.json]: Do we need webconferencing ?

File: commons-exo-extension/package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "meeds-gamification-github",
```
Comment (ahamdi) [inline commons-exo-extension/package.json]: shouldn't it be exo-gamification-github ?

File: commons-exo-extension/.eslintrc.json (hunk 2 above)
Comment (rdenarie) [reply commons-exo-extension/.eslintrc.json]: done

File: commons-exo-extension/package.json (hunk 3 above)
Comment (rdenarie) [reply commons-exo-extension/package.json]: done

**Pull request merged**